lib/gel/config.rb

```diff
@@ -34,20 +31,11 @@ def []=(group = nil, key, value)
   def read
     result = {}
     if File.exist?(@path)
-      context = nil
       File.read(@path).each_line do |line|
         line.chomp!
-        if line =~ /\A(\S[^:]*):\z/
-          context = result[$1] = {}
-        elsif line =~ /\A  ([^:]*): (.*)\z/
-          context[$1] = $2
-        elsif line =~ /\A([^:]*): (.*)\z/
-          result[$1] = $2
-        elsif line =~ /\A\s*(?:#|\z)/
-          # comment / empty
-        else
-          raise Gel::Error::UnexpectedConfigError.new(line: line)
-        end
+        key, value = line.split(':')
+        next unless key
+        result[key.downcase] = value&.strip
```

Comment:
@matthewd I downcased to keep all settings consistent, also `strip` the value too. Without the null check `&` some tests fail, I think it's because somewhere we are using the default -- bundler version for the config where we run all our tests, is that how it's supposed to work?